subsys/bluetooth/shell/cap_initiator.c

```diff
@@ -79,10 +474,23 @@ static int cmd_cap_initiator(const struct shell *sh, size_t argc, char **argv)
 
 SHELL_STATIC_SUBCMD_SET_CREATE(cap_initiator_cmds,
 #if defined(CONFIG_BT_BAP_UNICAST_CLIENT)
-			       SHELL_CMD_ARG(discover, NULL, "Discover CAS",
-					     cmd_cap_initiator_discover, 1, 0),
+	SHELL_CMD_ARG(discover, NULL,
+		      "Discover CAS", cmd_cap_initiator_discover, 1, 0),
+	SHELL_CMD_ARG(unicast-start, NULL,
```

Comment:
is it possible to use stop/start_sine from the bap shell on multiple coordinated set devices? - also, if so, maybe we should be able to set the frequency to distinguish different channels

Reply:
Not currently. The current implementation of the `start_sine` command only support a single TXing stream.